packages/jest-core/src/plugins/UpdateSnapshotsInteractive.ts

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable local/ban-types-eventually */
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
No, fix the path in .eslintrc instead 🙂

Reply:
https://github.com/facebook/jest/blob/246b5d3878f1e567ab13bbd9772a04076d693f09/.eslintrc.js#L85

Reply:
Thanks 🙌🏻